manifests/database/postgresql.pp

```diff
@@ -59,8 +59,11 @@
     # Only install pg_trgm extension, if database it is actually managed by the module
     if $manage_database {
 
-      # get the pg contrib to use pg_trgm extension
-      class { '::postgresql::server::contrib': }
+      # from postgresql version 10 onwards, this extension is no longer inside the contrib package, but is being bundled with the postgresql package itself
+      if (versioncmp($postgres_version, '10') == -1) {
+        # get the pg contrib to use pg_trgm extension
+        class { '::postgresql::server::contrib': }
```

Comment:
Maybe while here we can switch to:
```suggestion
        include postgresql::server::contrib
```
I remember having been beaten by this and reorder my profiles inclusions in a role when dealing with this module.